cocos/base/allocator/CCAllocatorStrategyGlobalSmallBlock.h

```diff
@@ -32,7 +32,7 @@
  Do not use Console::log or any other methods that use NEW inside of this
  allocator. Failure to do so will result in recursive memory allocation.
  ****************************************************************************/
-
+#include <string.h>
```

Comment:
Why have to include <string.h> in here?

Reply:
Becouse this header included in the CCAllocatorGlobalNewDelete.cpp that try to get the function 'memset' but can't whithout <string.h>.
Thank you